### ISSUE TITLE
Img2img enabling Controlnet has no effect on the posture

### ISSUE THREAD

**Issue by johnmark2030** [Mikubill/sd-webui-controlnet#1055]
The img2img enabling Controlnet has no effect on the posture - enabling it or not gives exact same image generated.

Prompt details:
Prompt: girl looking aside
Negative prompt: paintings, sketches, (worst quality:2), (low quality:2), (normal quality:2), cgi, lowres, normal quality, ((monochrome)), ((grayscale)), skin spots, acnes, skin blemishes, age spot, glans, nude, bad eye

lowres,bad anatomy,bad hands,text,error,missing fingers,extra digit,fewer digits,cropped,worst quality,low quality,normal quality,jpeg artifacts,signature,watermark,username,blurry,missing arms,long neck,Humpbacked,poorly drawn,malformed limbs,poorly drawn hands,mutilated,missing legs,unclear eyes,deformed

Steps: 30, Sampler: DPM++ SDE Karras, CFG scale: 7, Seed: 4193710843, Size: 512x512, Model hash: cc6cb27103, Model: v1-5-pruned-emaonly, Denoising strength: 0.5, ControlNet-0 Enabled: True, ControlNet-0 Module: openpose, ControlNet-0 Model: control_v11p_sd15_openpose [cab727d4], ControlNet-0 Weight: 1, ControlNet-0 Guidance Start: 0, ControlNet-0 Guidance End: 1

No error msg:

ControlNet preprocessor location: F:\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
Loading model: control_v11p_sd15_openpose [cab727d4]
Loaded state_dict from [F:\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_v11p_sd15_openpose.pth]
Loading config: F:\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_v11p_sd15_openpose.yaml
ControlNet model control_v11p_sd15_openpose [cab727d4] loaded.
Loading preprocessor: openpose
preprocessor resolution = 512
100%|██████████████████████████████████████████████████████████████████████████████████| 16/16 [00:05<00:00,  3.17it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 16/16 [00:04<00:00,  3.38it/s]
Loading model from cache: control_v11p_sd15_openpose [cab727d4]████████████████████████| 16/16 [00:04<00:00,  3.43it/s]

Settings
![screenshot2](https://user-images.githubusercontent.com/131640863/234611975-4a6fca7c-6902-4c6b-a636-b89fdeeb2a67.PNG)

Please let me know if any further information required.

**Comment by johnmark2030** [comment]
Just realize "openpose" is not working for the inputs I had.  It works using other controlnet models.

**Comment by Pauweltje** [comment]
Hi, where can I download: control_v11p_sd15_openpose [cab727d4] etc..
